docs/openapi/paths/api@services@{service_id}@pages@{pages_uuid}.yaml

```diff
@@ -0,0 +1,35 @@
+get:
+  tags:
+    - Default Text
+  summary: Get default text
+  description: Gets the default text for a page
+  operationId: default-text
+  parameters:
+  - name: service_id
+    in: path
+    description: The Service ID that needs to be updated
+    required: true
+    schema:
+      type: string
+  - name: page_uuid
+    in: path
+    description: The Page UUID
+    required: true
+    schema:
+      type: string
+  security:
+    - jwt: []
```

Comment:
at this point we won't be using jwt as the security mechanism. i am unsure how to represent the session type of security in openapi . . .

Reply:
perhaps i can change to `basicAuth: []` ?